charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -31,8 +31,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 69.8.2
```

Comment:
please bump the major version and leave a note in README.md about this. This is necessary for each CRD changes.